dist/robot.txt

```diff
@@ -0,0 +1,7 @@
+User-agent: Googlebot
+Disallow:
```

Comment:
`Disallow` need at least one entry, _empty_  is not valid

Reply:
Thanks for this catch, I removed this line